lapack/native/dlasq5.go

```diff
@@ -28,7 +28,8 @@ func (impl Implementation) Dlasq5(i0, n0 int, z []float64, pp int, tau, sigma fl
 		emin = z[j4+4]
 		d := z[j4] - tau
 		dmin = d
-		dmin1 = -z[j4]
+		// In the reference there are code paths that actually return this value.
```

Comment:
In the `else` branches for non-IEEE arithmetic there is a return before `dmin1` is set again.